relayer/crates/starknet-integration-tests/src/contexts/chain_driver.rs

```diff
@@ -45,6 +47,7 @@ delegate_components! {
         ErrorRaiserComponent: UseDelegate<HandleStarknetChainError>,
         RuntimeTypeProviderComponent: WithType<HermesRuntime>,
         RuntimeGetterComponent: WithField<symbol!("runtime")>,
+        ChainStartupWaiterComponent: WaitChainReachHeight<1>,
```

Comment:
I noticed that now `starknet-devnet` takes some seconds to bootup. So, I added this to wait for `starknet` blocks to produce...